### ISSUE TITLE
Changelog is not rendered on github

### ISSUE THREAD

**Issue by egoroof** [nodejs/nodejs.org#5533]
### URL:

https://nodejs.org/en

### Browser Name:

chrome

### Browser Version:

114

### Operating System:

windows 11

### How to reproduce the issue:

1. On https://nodejs.org/en go to link to the changelog of nodejs 18
2. Now you are on github markdown reader, but it does not rendered because of "This blob took too long to generate."

![image](https://github.com/nodejs/nodejs.org/assets/2605831/85de4d87-04af-4822-821f-8163386c75ac)

Maybe you should use your own place to keep changelogs? For example on the nodejs.org ?

**Comment by ovflowd** [comment]
I don't think this is a bug (at least on the Website) or something for the Website team. The Noe.js Releasers (cc @nodejs/releasers) should decide if either the release should have a separate `CHANGELOG` file or if we could upload it somewhere else.

Moving this Issue to `nodejs/node` as it makes more sense there.
